src/lib/y2network/connection/ethernet.rb

```diff
@@ -0,0 +1,8 @@
+require "y2network/connection/connection"
+
+module Y2Network
+  module Connection
+    class Ethernet < Connection
```

Comment:
this for me looks like that connection is too much ethernet, I think there are connections that has differences to Ethernet config.